pkg/login/social/azuread_oauth.go

```diff
@@ -97,7 +98,11 @@ func extractEmail(claims azureClaims) string {
 
 func extractRole(claims azureClaims) models.RoleType {
 	if len(claims.Roles) == 0 {
-		return models.ROLE_VIEWER
+		if setting.AutoAssignOrgRole == "" {
```

Comment:
I do believe this is always set and couldn't be empty so no need for this check.

Reply:
I see, will fix it